common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -622,14 +626,6 @@ private static boolean hasUnsafe0() {
         }
     }
 
-    private static long arrayBaseOffset0() {
-        if (!hasUnsafe()) {
-            return -1;
-        }
-
-        return PlatformDependent0.arrayBaseOffset();
```

Comment:
@normanmaurer what? :P I removed this method because I am storing the offset in a static final field in `PlatformDependent0`, because invoking the method directly is super slow.